addons/website_livechat/models/mail_channel.py

```diff
@@ -32,16 +33,19 @@ def channel_info(self, extra_info=False):
         for channel in self:
             visitor = channel.livechat_visitor_id
             if visitor:
-                channel_infos_dict[channel.id]['visitor'] = {
-                    'name': visitor.display_name,
-                    'country_code': visitor.country_id.code.lower() if visitor.country_id else False,
-                    'country_id': visitor.country_id.id,
-                    'is_connected': visitor.is_connected,
-                    'history': self.sudo()._get_visitor_history(visitor),
-                    'website': visitor.website_id.name,
-                    'lang': visitor.lang_id.name,
-                    'partner_id': visitor.partner_id.id,
-                }
+                try:
+                    channel_infos_dict[channel.id]['visitor'] = {
+                        'name': visitor.display_name,
+                        'country_code': visitor.country_id.code.lower() if visitor.country_id else False,
+                        'country_id': visitor.country_id.id,
+                        'is_connected': visitor.is_connected,
+                        'history': self.sudo()._get_visitor_history(visitor),
+                        'website': visitor.website_id.name,
+                        'lang': visitor.lang_id.name,
+                        'partner_id': visitor.partner_id.id,
+                    }
+                except AccessError:
+                    continue
```

Comment:
`pass` would be more clear here, as we really want do "do nothing" (`continue` wouldn't work if we had/need to have code after this try/catch block)